optimism/src/keccak/column.rs

```diff
@@ -96,9 +96,52 @@ impl FoldingColumnTrait for Column {
 /// The row is split into the following entries:
 /// - hash_index: Which hash this is inside the circuit
 /// - step_index: Which step this is inside the hash
-/// - mode_flags: Round, Absorb, Squeeze
-/// - curr: Contains 1969 witnesses used in the current step including Input and RoundConstants
+/// - mode_flags: Round, Absorb, Squeeze, Root, PadLength, InvPadLength, TwoToPad, PadBytesFlags, PadSuffix, RoundConstants
+/// - curr: Contains 1969 witnesses used in the current step including Input
 /// - next: Contains the Output
+///
+///   Keccak Witness Columns: KeccakWitness.cols
+///  ----------------------------------------------
+/// | 0 | 1 | 2 | 3..154 | 155..2119 | 2120..2219 |
+///  ----------------------------------------------
+///   0 -> hash_index
+///   1 -> block_index
+///   2 -> step_index
+///   3..154 -> mode_flags
+///          -> 3: FlagRound
+///          -> 4: FlagAbsorb
+///          -> 5: FlagSqueeze
+///          -> 6: FlagRoot
+///          -> 7..142: PadBytesFlags
+///          -> 143: PadLength
+///          -> 144: InvPadLength
+///          -> 145: TwoToPad
+///          -> 146..150: PadSuffix
+///          -> 151..154: RoundConstants
+///   155..2123 -> curr
+///         155                                                                        2119
+///          <--------------------------------if_round<---------------------------------->
+///          <-------------if_sponge------------->    
+///         155                                 954           
+///          -> SPONGE:                          | -> ROUND:    
+///         -> 155..254: Input == SpongeOldState | -> 155..254: Input == ThetaStateA    
+///         -> 255..354: SpongeNewState          | -> 255..334: ThetaShiftsC           
+///                    : 323..354 -> SpongeZeros | -> 335..354: ThetaDenseC
```

Comment:
Why is this `SpongeZero` part aligned differently?

Reply:
It refers to the last 32 bytes of the new state which are supposed to be zeros.